includes/pet-details-form.php

```diff
@@ -169,3 +170,5 @@
         </svg>
     </span>
 </p>
+<button>Submit</button>
+</form>
```

Comment:
I was thinking we'd put the form tags and buttons on the pages that pull in this include. That way we can have different actions per page